sensenet/layers/extract.py

```diff
@@ -6,10 +6,12 @@ def add(config, layer):
 
 
 def activation(config, layer):
-    return {
-        "type": "activation",
-        "activation_function": layer.activation.__name__,
-    }
+    if "negative_slope" in config and "max_value" in config:  # ReLU layer
+        afn = "relu"
+    else:
+        afn = layer.activation.__name__
+
+    return {"type": "activation", "activation_function": afn}
```

Comment:
Is this something that can help me with my never ending plates project?

Reply:
> Is this something that can help me with my never ending plates project?

Nope!  Unrelated.